modules/database/src/ioc/db/dbChannel.h

```diff
@@ -9,14 +9,16 @@
 * in file LICENSE that is included with this distribution.
 \*************************************************************************/
 
-/*
+#ifndef INC_dbChannel_H
+#define INC_dbChannel_H
+
+/** \file dbChannel.h
+ * @brief Declarations fir the @ref dbChannel "dbChannel" record type
```

Comment:
From a quick look at the code, it seems that Doxygen `\cmd` instead of `@cmd` are preferred. It might need official confirmation, though.

Reply:
Some editors may recognize one or the other, some files come from other projects and already had annotations, so we have both `@file` and `\file` in the tree and don't really care which get used since Doxygen understands both.